docs/reference/commandline/index.md

```diff
@@ -182,3 +182,12 @@ read the [`dockerd`](dockerd.md) reference page.
 | [plugin push](plugin_push.md) | Push a plugin to a registry                  |
 | [plugin rm](plugin_rm.md) | Remove a plugin                                  |
 | [plugin set](plugin_set.md)  | Change settings for a plugin                  |
+
+### System commands
+
+| Command | Description                                                        |
+|:--------|:-------------------------------------------------------------------|
+| [system df](system_df.md) | Show docker filesystem usage                      |
+| [system events](events.md) | Get real time events from the server            |
+| [system info](info.md) | Display system-wide information                     |
+| [system prune](system_prune.md) | Delete unused data                         |
```

Comment:
I  think that commands here have some conflict with https://github.com/docker/docker/blob/master/docs/reference/commandline/index.md#docker-management-commands, WDYT? @vdemeester @thaJeztah